doc/user/content/third-party/supported-tools.md

```diff
@@ -23,11 +23,13 @@ Beta-level support means that there may be small performance issues and minor mi
 | ----------------------- | -------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
 | [dbt](/third-party/dbt) | Create views and materialized views in your Materialize instance using dbt | Full `dbt-materialize` adapter support is a [work in progress](https://github.com/MaterializeInc/materialize/issues/5462). |
 | [Redpanda](/third-party/redpanda) | Set up Redpanda as a data source. | Kafka offsets are ignored.
+| [Metabase](/third-party/metabase) | Create business intelligence dashboards on top of your Materialize data | Connect Metabase to Materialize using the **PostgreSQL** database type. See [using Metabase](/third-party/metabase) page for more details.
 
+<!--
 ## Alpha-level support
 
 Alpha-level support means that some of our community members have made this integration work, but we haven't tested it ourselves and can't guarantee its stability.
 
 | Tool | Purpose | What's missing? |
 |------|---------|---------|
-| [Metabase](/third-party/metabase) | Create business intelligence dashboards on top of your Materialize data | Running Metabase cleanly, without our forked changes, requires further [pgjdbc support](https://github.com/MaterializeInc/materialize/issues/3727).
+-->
```

Comment:
I left the "Alpha-level Support" here, albeit commented out, so as not to lose visibility that we have this classification.